jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -0,0 +1,248 @@
+- Server v3.x
+- Server v2.x
+---
+= Test splitting to speed up your pipelines
+:page-layout: classic-docs
+:page-liquid:
+:page-description: "Learn to split and run tests in parallel in CircleCI"
+:icons: font
+:toc: macro
+:toc-title:
+
+This guide walks you through a basic example of test splitting in CircleCI. 
+
+toc::[]
+
+== Introduction
+
+Test splitting is a technique in which tests are executed simultaneously across multiple execution environments. Test splitting takes advantage of a feature called <<parallelism-faster-jobs#,_parallelism_>>, which allows you to run a job in CircleCI on different nodes at the same time.
+
+You may have a test suite that consists of dozens or hundreds of tests, and executing them one after the other can take up a lot of time and consume a lot of credits. When you split tests, you have the opportunity to significantly reduce wait times, receive feedback more quickly, and optimize your plan usage. 
+
+Test splitting in CircleCI can work with many testing frameworks, including Jest, pytest, Maven, and Gradle, just to name a few. 
+
+In this tutorial, you will:
+
+* Set up a basic React app as a project in CircleCI.
+* Modify the project's `.circleci/config.yml` file to split tests based on timing data.
+* View the resulting parallel test runs in the CircleCI web app.
+* See how test splitting can help decrease pipeline run times and optimize credit usage.
+
+To complete this tutorial, you need:
+
+* A CircleCI account - if you do not have an account, <<first-steps#,you can sign up for free>>.
+* A Version Control System (VCS) provider, such as GitHub or BitBucket, connected to your CircleCI account. If you have not already done so, follow the steps in the <<gh-bb-integration#,GitHub and Bitbucket Integration>> page to connect your VCS provider.
+
+== About the sample app
+
+You will use a basic React app for this tutorial. The project repository is available on https://github.com/ryanpedersen42/circleci-react-test-splitting[GitHub]. The app was created using https://create-react-app.dev/[Create React App], and is set up to use the https://jestjs.io/[Jest] testing framework. It uses the https://github.com/jest-community/jest-junit[jest-junit] reporter to export test results as JUnit XML files. 
+
+When you set up the project in CircleCI later in this tutorial, you'll select the option to use an existing config.yml that you can edit. The template used with this tutorial is a starter configuration that can be used with Node projects. Read the following section for a quick walkthrough of the configuration, or feel free to skip ahead to set up the project if you're already familiar with setting up Node projects in CircleCI.
+
+=== Configuration walkthrough
+
+* This project uses the configuration `version: 2.1`. This enables CircleCI features such as orbs and resuable configuration. More information on config version 2.1 can be found in the <<configuration-reference#,Configuring CircleCI>> reference document.
```

Comment:
Can you put a short paragraph at the start of this section to link to the configuration file you are talking about?

Reply:
Or a snippet of the full config?